src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -201,6 +201,7 @@ public static class Captions
       public static readonly string ChartExportOptions = "Chart Export Options";
       public static readonly string No = "No";
       public static readonly string Yes = "Yes";
+      public static readonly string ReallyRemoveObservedDataFromSimulation = $"Really remove {ObjectTypes.ObservedData} from simulation?\nHint: {ObjectTypes.ObservedData} will not be deleted from project";
```

Comment:
I think it should be "... from **the** simulation" and "...from **the** project"

Reply:
done